src/celengine/solarsys.cpp

```diff
@@ -39,15 +39,6 @@
 using namespace Eigen;
 using namespace std;
 
-
-enum Disposition
```

Comment:
Definitely no.

either:
```
location->loadCategories(objectData, static_cast<CatEntry::Dispositon>disposition, directory);
```
or
add a general `celengine/disposition.h` with `enum Disposition` and use it instead (instead of CatEntry::Dispositon too). In this case make 1st commit the change above and another commit with this chnage.

Reply:
Even if we assume that CatEntry may state for both Category Entry and Catalog Entry (or we'll just rename it to something more general)?

Reply:
I don't get your question :) Nie rozumiem :)

Reply:
Casting looks extremely ugly, while creating another header file looks a bit excessively. So, why not `CatEntry::DataDisposition`, if not because of its name and suggested exclusive function?

Reply:
Because CatEntry add unneeded and not fully correct dependency between different modules. If you don't like to add an new .h, you can check if it's possible to add the definition to some existing headers.